bin/sku.js

```diff
@@ -13,9 +15,38 @@ if (args.debug) {
   debug.enable(process.env.DEBUG);
 }
 
+let skipConfigure = false;
+let skipValidatePeerDeps = false;
+const packageJson = getPathFromCwd('./package.json');
+const packageJsonExists = fs.existsSync(packageJson);
+
+if (packageJsonExists) {
+  const {
+    skuSkipConfigure = false,
+    skuSkipValidatePeerDeps = false,
+  } = require(packageJson);
+  skipConfigure = skuSkipConfigure;
+  skipValidatePeerDeps = skuSkipValidatePeerDeps;
+}
+
 const configureProject = async () => {
-  const configure = require('../lib/configure');
-  await configure();
+  if (!skipConfigure) {
+    const configure = require('../lib/configure');
+    await configure();
+  } else {
+    log(`"skuSkipConfigure" set in ${packageJson}, skipping sku configuration`);
+  }
```

Comment:
Not a fan of early exits [like over here](https://github.com/seek-oss/sku/blob/aeeb5a7e59c3ce75670911c6c5216aab57dddc47/config/vocab/vocab.js#L6-L9)?

```suggestion
  if (skipConfigure) {
    log(`"skuSkipConfigure" set in ${packageJson}, skipping sku configuration`);
    return;
  }
  const configure = require('../lib/configure');
  await configure();
```